aya_tamimi/task.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+
+class Task :
+    def __init__(self ,title,description,due_date ,status ):
+        self.task_id = str(uuid.uuid4())
+        self.title = title
+        self.description = description
+        self.due_date =due_date
+        self.status = status
+    
+
+    def update_details(self, title=None, description=None, due_date=None, status=None):
+        if title is not None:
+            self.title = title
+        if description is not None:
+            self.description = description
+        if due_date is not None:
+            self.due_date = due_date
+        if status is not None:
+            self.status = status
+
+       
+    def display(self):
+        return (
+            f"Task Details:\n"
+            f"-----------------------------\n"
+            f"Task Id     : {self.task_id}\n"
+            f"Title       : {self.title}\n"
+            f"Status      : {self.status}\n"
+            f"Description : {self.description}\n"
+            f"Due Date    : {self.due_date}\n"
+        )
```

Comment:
you can do it in a better way 
 
 like 
` return (f"Task Id     : {self.task_id}\n Title       : {self.title}\n")`